gateway/src/node/lp_listener/mod.rs

```diff
@@ -186,6 +315,29 @@ pub struct LpHandlerState {
 
     /// LP configuration (for timestamp validation, etc.)
     pub lp_config: LpConfig,
+
+    /// In-progress handshakes keyed by session_id
```

Comment:
comment rot. it's keyed by receiver index which is no longer computed deterministically from the keys but rather proposed by the initiator